experimental-statement.md

```diff
@@ -0,0 +1,21 @@
+# Experimental Statement
```

Comment:
I'm reluctant to introduce this file in all of our projects. I think we'll have an easier time maintaining these projects linking out to one centralized copy.

Reply:
Moved to external file.